modules/virt-about-resource-quota-limits.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-specifying-nodes-for-vms.adoc
```

Comment:
Please update this to the new assembly path that you created

Reply:
Doh! Thanks.